etc/vagrant/.gitignore

```diff
@@ -0,0 +1 @@
+.vagrant
```

Comment:
I don't have a preference - thought it would be more hermetic to keep it near the Vagrantfile. Happy to move if you think it makes sense.